bin/cron/sync_dropbox

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+
+# This syncs content changes from the shared dropbox folder to our github repo
+# on the staging machine. This is required for contents changes from dropbox to
+# be deployed.
+
+require_relative '../../lib/cdo/only_one'
+exit unless only_one_running?(__FILE__)
+require_relative '../../dashboard/config/environment'
+exit unless rack_env?(:staging) && CDO.dashboard_hostname == 'staging-studio.code.org'
```

Comment:
I think we don't want this check since requiring `dashboard/config/environment` is what makes the script take such a long time to run. Then we can get it running every 5-10 seconds.

Reply:
Will has recommended including `deployment.rb` instead - this gives access to the CDO variables/secrets without loading anything rails.